internal/resources/schemas/lb_profile_schemas.go

```diff
@@ -0,0 +1,450 @@
+					Optional:         true,
+					Default:          "INSERT",
+					Description:      "Network Loadbalancer Supported values are `INSERT`,`PREFIX`, `REWRITE`",
+				},
+				"cookie_type": {
+					Type:             schema.TypeString,
+					ValidateDiagFunc: validations.StringInSlice([]string{"LBPersistenceCookieTime", "LBSessionCookieTime"}, false),
+					Optional:         true,
+					Default:          "LBSessionCookieTime",
+					Description:      "Network Loadbalancer Supported values are `LBPersistenceCookieTime`,`LBSessionCookieTime`",
+				},
+				"cookie_domain": {
+					Type:        schema.TypeString,
+					Optional:    true,
+					Description: "cookie_domain for Network Load balancer Profile",
+				},
+				"cookie_path": {
+					Type:        schema.TypeString,
+					Optional:    true,
+					Description: "cookie_path for Network Load balancer Profile",
+				},
+				"max_idle_time": {
+					Type:        schema.TypeInt,
+					Optional:    true,
+					Description: "max_idle_time for Network Load balancer Profile",
+				},
+				"max_cookie_age": {
+					Type:        schema.TypeInt,
+					Optional:    true,
+					Description: "max_cookie_age for Network Load balancer Profile",
+				},
+				"share_persistence": {
+					Type:        schema.TypeBool,
+					Optional:    true,
+					Description: "ntlm_authentication for Network Load balancer Profile",
+				},
+			},
+		},
+	}
+}
+
+func SourceIPProfileSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeList,
+		Optional:    true,
+		Description: "SourceIP Profile configuration",
+		MaxItems:    1,
+		ExactlyOneOf: []string{"http_profile", "tcp_profile", "udp_profile",
+			"cookie_profile", "sourceip_profile", "generic_profile",
+			"client_profile", "server_profile"},
+		ConflictsWith: []string{"http_profile", "tcp_profile",
+			"udp_profile", "cookie_profile", "generic_profile",
+			"client_profile", "server_profile"},
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"profile_type": {
+					Type: schema.TypeString,
+					ValidateDiagFunc: validations.StringInSlice([]string{
+						"application-profile", "ssl-profile", "persistence-profile",
+					}, false),
+					Optional:    true,
+					Default:     "persistence-profile",
+					Description: "Network Loadbalancer Supported values are `application-profile`, `ssl-profile`, `persistence-profile`",
+				},
+				"share_persistence": {
+					Type:        schema.TypeBool,
+					Optional:    true,
+					Default:     false,
+					Description: "ntlm_authentication for Network Load balancer Profile",
+				},
+				"ha_persistence_mirroring": {
+					Type:        schema.TypeBool,
+					Optional:    true,
+					Default:     false,
+					Description: "ha_persistence_mirroring for Network Load balancer Profile",
+				},
+				"persistence_entry_timeout": {
+					Type:             schema.TypeInt,
+					Optional:         true,
+					Default:          300,
+					ValidateDiagFunc: validations.IntBetween(1, 2147483647),
```

Comment:
can Int have a value like 2147483647

Reply:
max int32   = 2147483647 you can see this link https://stackoverflow.com/questions/6878590/the-maximum-value-for-an-int-type-in-go